sites/partners/page_content/locale_overrides/general.json

```diff
@@ -228,7 +228,8 @@
   "listings.lottery.exportFile": "This file includes the lottery raw rank and preferences data for all applications.",
   "listings.lottery.exportFileNoPreferences": "This file includes the lottery raw rank for all applications.",
   "listings.lottery.history": "History",
-  "listings.lottery.historyLogAutomatic": "by Property",
+  "listings.lottery.historyLogByProperty": "by Property",
```

Comment:
Can check w product but I think we only want one of these used for anything we did automatically

Reply:
I did check with Sarah and she said "I think for closing, it should say "by Property". If the results were auto published by the system, it should say "by System""